docker-compose.example.yml

```diff
@@ -37,32 +37,6 @@ services:
       - ./photos_path:/photos:ro
       - api_cache:/app/cache
 
-  ui:
-    build:
-      context: ./ui
-      args:
-        # Change This: The publicly exposed url for the graphql api
-        # For example if the server is available from the domain example.com,
-        # change this value to http://example.com/api/graphql
-        GRAPHQL_ENDPOINT: http://localhost:8080/api/graphql
```

Comment:
This variable is used when building the ui to know where to connect to the backend see `apolloClient.js` below.

https://github.com/viktorstrate/photoview/blob/e0ada51b6ba0a787ced22d309a100e6848c3641b/ui/src/apolloClient.js#L16-L21

This could probably be easily fixed, as the front-end and back-end are served on the same domain

Reply:
This has been resolved in 48520a7